### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by rehanmohiuddin**
- Added & Revamped Login,Register UI
- Added Reusable Button
- Added Reusable Loader

### CODE REVIEW

File: src/Components/Form/index.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                    ? "input error-border"
+                    : "input"
+                }
+              >
+                <FontAwesomeIcon size="1x" icon={icon} />
+                <input
+                  name={key}
+                  onChange={(e) => {
+                    onTextInput(key, e.target.value);
+                    // textInput(key, e.target.value);
```
Comment (Prabhav-Sharma) [inline src/Components/Form/index.jsx:34]: Can remove comment, if not needed

**Review by Prabhav-Sharma — APPROVED** [review]

File: src/Components/Button/index.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+      <div onClick={callBack} className={"btn " + getBtnType[style]}>
+        {children ? children : title}
+      </div>
+    ),
+    [BUTTON.LINK]: (
+      <Link className={getBtnType[style]} to={linkTo}>
+        {title}
+      </Link>
+    ),
+  };
```
Comment (mariyasada) [inline src/Components/Button/index.jsx:31]: this is new for me.

File: src/Pages/Home/index.scss [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+@import "../../styles/variables.scss";
+
+.home-container {
+  @extend .flex-row;
+  width: 100%;
+  .home-header {
+    width: 100%;
+    @extend .flex-row, .flex-end, .padding-full;
+  }
+  .list {
```
Comment (mariyasada) [inline src/Pages/Home/index.scss:11]: this is also good

**Review by mariyasada — APPROVED** [review]
the code structure is good. liked that you have used a preprocessor. learning some new things. like reusable button and @extend also.Th is PR is good to merge

**Pull request merged**